podpac/core/data/datasource.py

```diff
@@ -369,12 +399,11 @@ def eval(self, coordinates, output=None):
         if self.coordinates.crs.lower() != coordinates.crs.lower():
             coordinates = coordinates.transform(self.coordinates.crs)
 
-        # intersect the coordinates with requested coordinates
-        # to get coordinates within requested coordinates bounds
+        # intersect the coordinates with requested coordinates to get coordinates within requested coordinates bounds
         # TODO: support coordinate_index_type parameter to define other index types
-        (self._requested_source_coordinates, self._requested_source_coordinates_index) = self.coordinates.intersect(
-            coordinates, outer=True, return_indices=True
-        )
+        (rsc, rsci) = self.coordinates.intersect(coordinates, outer=True, return_indices=True)
```

Comment:
Does this need to know the boundary data?

Reply:
I passed the boundary through at one point, but it does not seem to need it. Maybe as an optimization.